### DISCUSSION TITLE
pls give a Screenshots From your Setup Buttons on streamdeck

### DISCUSSION THREAD

**Discussion by rug4ever** [nguyenquyhy/Flight-Tracker-StreamDeck#89] [Q&A] [answered]
I have a many Buttons Programm with many hour to inverst. 

I can give Screenshots with the Infos From the streamdeck Button variable. everywhere have interessting From the Screenshot? 

maybe want to send everywhere himself Button Setup From streamdeck. 

I think have a complete Setup From c172 garmin 1000 

or give everywhere the Setups Profil copy share here.

**Comment by rmroc451** [comment]
I'm going to move this into the new discussion area, since there really isn't anything actionable on this.

**Reply by kranberi** [reply]
Im totally new to this . ive gotten everything set up and ive tried the prebuilt profiles for c172 g1000 etc.. It works great.

Is it possible to do basic profiles for airliners? a320? etc?

**Comment by Andywmm9** [comment]
![image](https://user-images.githubusercontent.com/45742070/103393607-1b40d080-4ae9-11eb-9dc7-80f4ef9295bd.png)

**Comment by nguyenquyhy** [comment, marked as answer]
RMROC has created a discussion for profile sharing #128, so if anyone here has useful profiles, please share an export in the discussion thread #128.

Since version 0.23, you can also embed ON/OFF images directly in the profile, so your export can work on any other computers.
